mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -6792,6 +6812,16 @@
 													"    pm.expect(tv4.missing.length).to.equal(0, \"Missing schemas: \" + JSON.stringify(tv4.missing));",
 													"});",
 													"",
+													"//Check that tagList is not empty",
+													"pm.test('Creating tagList', function(){",
+													"    pm.expect(response.data.attributes.tags.tagList.size) !== 0;",
+													"});",
+													"    ",
+													"//Check that tagList is not empty",
+													"pm.test('Creating tagList', function(){",
+													"    pm.expect(response.data.attributes.tags.tagList[0]).eq(\"foo-tag\");",
+													"});",
```

Comment:
Similar to above, check of taglist should be done after response.data has been confirmed and it has been confirmed that tags attribute is present (around line 6842)